### PR TITLE
docs(troubleshooting): add advanced prometheus investigation

### PR DESCRIPTION
**Pull request by sumo-drosiek**
Signed-off-by: Dominik Rosiek <drosiek@sumologic.com>

##### Description

Fill in your description here.

---

##### Checklist

- [ ] Changelog updated

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by kkujawa-sumo — APPROVED** [review]

File: deploy/docs/Troubleshoot_Collection.md [hunk 1]
```diff
@@ -262,6 +262,41 @@ kubectl port-forward collection-sumologic-xxxxxxxxx-xxxxx 8080:24231
...
+     serviceAccountName: <service account name used by prometheus (e.g. collection-kube-prometheus-prometheus)>
+   EOF
+   ```
+
+2. Go into the container:
+
+   ```bash
+   kubectl exec -it sumologic-debug -n <namespace> bash
+
+3. Talk with API directly like prometheus does, e.g.
```
Comment (kkujawa-sumo) [inline deploy/docs/Troubleshoot_Collection.md:294]: I don't like "talk" here but it could stay :P

Comment (sumo-drosiek) [inline deploy/docs/Troubleshoot_Collection.md:294]:
```suggestion
3. Query API directly like prometheus does, e.g.
```

Comment (sumo-drosiek) [reply deploy/docs/Troubleshoot_Collection.md]: Is it better now?

File: deploy/docs/Troubleshoot_Collection.md [hunk 2]
```diff
@@ -262,6 +262,41 @@ kubectl port-forward collection-sumologic-xxxxxxxxx-xxxxx 8080:24231
...
+
+2. Go into the container:
+
+   ```bash
+   kubectl exec -it sumologic-debug -n <namespace> bash
+
+3. Query API directly like prometheus does, e.g.
+
+   ```bash
+   curl https://10.0.2.15:10250/metrics/cadvisor --insecure --cacert /var/run/secrets kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)"
```
Comment (sumo-drosiek) [inline deploy/docs/Troubleshoot_Collection.md:297]:
```suggestion
   curl https://10.0.2.15:10250/metrics/cadvisor --insecure --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)"
```

File: docs/troubleshoot-collection.md [hunk 3]
```diff
@@ -182,8 +218,8 @@ You can also check `prometheus_remote_storage_.*` metrics to look for success/fa
 ### Missing metrics - cannot see cluster in Explore
 
 If you are not seeing metrics coming in to Sumo or/and your cluster is not showing
-up in [Explore][explore] it is most likely due to the fact that Prometheus pod
-is not running.
+up in [Explore](https://help.sumologic.com/docs/observability/kubernetes/monitoring#open-explore)
+it is most likely due to the fact that Prometheus pod is not running.
```
Comment (sumo-drosiek) [inline docs/troubleshoot-collection.md:222]: I have no idea what was it about, but markdownlint was complaining

**Pull request merged**